client/src/app/app.module.ts

```diff
@@ -21,56 +23,52 @@ import { MatSnackBarModule } from '@angular/material/snack-bar';
 import { MatToolbarModule } from '@angular/material/toolbar';
 import { MatTooltipModule } from '@angular/material/tooltip';
 
-import { BrowserModule } from '@angular/platform-browser';
-import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
-
-import { AppRoutingModule } from './app-routing.module';
+import { AddUserComponent } from './users/add-user.component';
 import { AppComponent } from './app.component';
+import { AppRoutingModule } from './app-routing.module';
 import { HomeComponent } from './home/home.component';
-import { AddUserComponent } from './users/add-user.component';
 import { UserCardComponent } from './users/user-card.component';
 import { UserListComponent } from './users/user-list.component';
 import { UserProfileComponent } from './users/user-profile.component';
 import { UserService } from './users/user.service';
 
 const MATERIAL_MODULES: any[] = [
-  MatListModule,
   MatButtonModule,
-  MatIconModule,
-  MatToolbarModule,
   MatCardModule,
-  MatMenuModule,
-  MatSidenavModule,
-  MatInputModule,
+  MatDividerModule,
   MatExpansionModule,
-  MatTooltipModule,
-  MatSelectModule,
-  MatOptionModule,
   MatFormFieldModule,
-  MatDividerModule,
+  MatIconModule,
+  MatInputModule,
+  MatListModule,
+  MatMenuModule,
+  MatOptionModule,
   MatRadioModule,
-  MatSnackBarModule
+  MatSidenavModule,
+  MatSelectModule,
+  MatSnackBarModule,
+  MatToolbarModule,
+  MatTooltipModule,
 ];
 
 @NgModule({
```

Comment:
I also tried to alphabetize elsewhere. @floogulinc suggests (correctly) that order can matter for these, so I'm going to make another commit that puts back the dangerous things into the order they were in before. This didn't seem to cause problems, but it was an arbitrary decision just meant to help readability. If it's not safe to do, better to leave it confusing.